programs/storage_api/src/storage_processor.rs

```diff
@@ -252,11 +252,11 @@ mod tests {
         let (genesis_block, mint_keypair) = create_genesis_block(1000);
         let mint_pubkey = mint_keypair.pubkey();
         let replicator_keypair = Keypair::new();
-        let replicator = replicator_keypair.pubkey();
+        let replicator_pubkey = replicator_keypair.pubkey();
```

Comment:
I've already handled this test in my PR. https://github.com/solana-labs/solana/pull/4377 this might be the only conflict. But I like your method of doing this over mine (I added keypairs for the nodes). Let it conflict. I'll rebase.